apps/scan/frontend/src/screens/scan_error_screen.tsx

```diff
@@ -78,19 +146,21 @@ export function ScanErrorScreen({
       voterFacing
     >
       <FullScreenPromptLayout
-        title={appStrings.titleScannerBallotNotCounted()}
+        title={title}
         image={
           <FullScreenIconWrapper>
             <Icons.Delete color="danger" />
           </FullScreenIconWrapper>
         }
       >
         <P>{errorMessage}</P>
```

Comment:
I think we should always bold this line for consistency and readability.

Reply:
3be03c2c450db11023f0ef0070eed9ff7900baa3